config/initializers/assets.rb

```diff
@@ -0,0 +1 @@
+Rails.application.config.assets.precompile += %w( print.css I18n.css sidebar.js dashboard.css dashboard.js dragdrop.js methods.js post.js comment_expand.js leaflet-blurred-location/dist/Leaflet.BlurredLocation.js dynamic_search.js notes.js textbox_expand.js question.js )
```

Comment:
Will this need to be maintained as we add new assets?

Reply:
If so, let's add a comment to `/app/assets/javascripts/application.js` and  `/app/assets/javascripts/application.css`

Reply:
Yeah it needs to be maintained in rails 5.0. I will add the comment to `/app/assets/javascripts/application.js` and `/app/assets/javascripts/application.css`

Reply:
This is covered in this commit: https://github.com/publiclab/plots2/pull/2726/commits/e7d451e7da833c7db63701ee822010e6f01f55a2. Please check the commit.